### DISCUSSION TITLE
ESLint 검사시간이 오래 걸려요

### DISCUSSION THREAD

**Discussion by caseBread** [Yangjaecheon-Otter-Guardians/simple-thumbnail#122] [Ideas]
### 주제

Features

### Body

기능에 비해 workflow 작동 시간이 오래 걸리는 것 같습니다.

다른분들 생각도 궁금하네요

### BDD TASK

```bash
- [ ] ESLint 검사 시간 30초 이내로 줄이기
...
```

**Comment by da-in** [comment]
우선 github actions의 ubuntu 18.04 지원 종료로 인하여 제대로 동작하지 않던 부분은, ubuntu-latest로 수정하여 메모 남겨둡니다.

**Reply by caseBread** [reply]
아하 어쩐지 안된다 싶었는데 우분투 지원종료가 되었군요 😞

22.04 버전이 가장 최신 LTS버전인듯한데 이거로 교체하는게 좋을까요?

**Reply by da-in** [reply]
우선 deploy workflows는 `ubuntu-latest`로 수정했고 제 기준 정상 동작은 확인했는데,  
discussion 남겨주셨던 것 보다 빠르게 동작하는지 체크 필요합니다.
- https://github.com/Yangjaecheon-Otter-Guardians/simple-thumbnail/blob/main/.github/workflows/gh-pages.yml
